pkg/version/version.go

```diff
@@ -49,6 +49,12 @@ var UpgradePaths = map[Components]Info{
 	{12, 0, 2}: {"v23.3.x", Components{23, 3, 0}},
 	{12, 0, 3}: {"v23.3.x", Components{23, 3, 0}},
 	{12, 0, 4}: {"v23.3.x", Components{23, 3, 0}},
+	// From here upwards the x.4.0 release (4th quarter release)
+	// is the LTS version so all releases will have the closest x.4.0
+	// as their next release they can be upgraded to.
+	// example:
+	//	- {23, 4, 0}: {"v24.4.x", Components{24, 4, 0}}
+	//	- {24, 1, 0}: {"v24.4.x", Components{24, 4, 0}}
```

Comment:
Can we add some UT to make sure we got the logic correctly? Some cases to confirm:
- 12.0.3 -> 12.0.4 (Allowed)
- 12.0.3 -> 23.3.0 (Allowed)
- 12.0.3 -> 23.4.0 (Allowed)
- 12.0.3 -> 24.4.0 (Blocked)
- 12.0.3 -> 24.1.0 (Blocked)
- 23.3.0 -> 23.4.0 (Allowed)
- 23.3.0 -> 24.2.0 (Blocked)
- 23.3.0 -> 24.3.0 (Blocked)